.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -20,6 +20,21 @@ body:
       description: What is the problem? A clear and concise description of the bug.
     validations:
       required: true
+  - type: checkboxes
+    id: regression
+    attributes:
+      label: Regression Issue
+      description:  What is a regression? If it worked in a previous version but doesn’t in the latest version, it’s considered a regression. In this case, Please provide specific version number in the report.
+      options:
+        - label: Select this option if this issue appears to be a regression.
+          required: false
+  - type: input
+    id: working-version
+    attributes:
+      label: Last Known Working CDK Version
+      description: Specify the last known CDK version where this code was functioning as expected(if applicable).
+    validations:
+      required: false
```

Comment:
For my learning purpose, how is `validations` different from 
```
options:
          required: false
```

Reply:
haven't seen this field options in template but validation for the text area field is enforced when it is a necessary input required by the user.

Reply:
Does that mean you're making the version field required?

Reply:
making the field not required, options are seems to be for an array field from the [documentation](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema#attributes-for-checkboxes)

Reply:
I see, thanks for the clarification. Good to approve once addressing Grace's feedback.